financial-connections/src/main/java/com/stripe/android/financialconnections/features/networkinglinkloginwarmup/NetworkingLinkLoginWarmupScreen.kt

```diff
@@ -84,181 +88,165 @@ private fun NetworkingLinkLoginWarmupContent(
+    onContinueClick: () -> Unit,
+    onSkipClicked: () -> Unit
+) {
+    Column {
+        FinancialConnectionsButton(
+            loading = false,
+            enabled = disableNetworkingAsync !is Loading,
+            type = Type.Primary,
+            onClick = onContinueClick,
+            modifier = Modifier
+                .semantics { testTagsAsResourceId = true }
+                .testTag("existing_email-button")
+                .fillMaxWidth()
+        ) {
+            Text(text = stringResource(id = R.string.stripe_networking_link_login_warmup_cta_continue))
+        }
+        Spacer(modifier = Modifier.size(16.dp))
+        FinancialConnectionsButton(
+            loading = false,
+            enabled = disableNetworkingAsync !is Loading,
+            type = Type.Secondary,
+            onClick = { onSkipClicked() },
+            modifier = Modifier
+                .semantics { testTagsAsResourceId = true }
+                .testTag("skip-button")
+                .fillMaxWidth()
+        ) {
+            Text(text = stringResource(id = R.string.stripe_networking_link_login_warmup_cta_skip))
+        }
+    }
 }
 
 @OptIn(ExperimentalComposeUiApi::class)
 @Composable
 internal fun ExistingEmailSection(
-    email: String,
-    onContinueClick: () -> Unit
+    email: String
 ) {
     Row(
         verticalAlignment = Alignment.CenterVertically,
         modifier = Modifier
             .fillMaxWidth()
             .semantics { testTagsAsResourceId = true }
-            .testTag("existing_email-button")
-            .clickableSingle { onContinueClick() }
-            .clip(RoundedCornerShape(8.dp))
+            .clip(RoundedCornerShape(12.dp))
             .border(
                 width = 1.dp,
-                color = FinancialConnectionsTheme.colors.borderDefault,
-                shape = RoundedCornerShape(8.dp)
+                color = v3Colors.border,
+                shape = RoundedCornerShape(12.dp)
             )
-            .padding(12.dp)
+            .padding(horizontal = 16.dp, vertical = 12.dp)
     ) {
-        Column(
-            Modifier.weight(1f)
+        Box(
+            contentAlignment = Alignment.Center,
+            modifier = Modifier
+                .size(32.dp)
+                .background(color = LinkColors.Brand200, shape = CircleShape)
         ) {
             Text(
-                text = stringResource(id = R.string.stripe_networking_link_login_warmup_email_label),
-                style = FinancialConnectionsTheme.typography.caption,
-                color = FinancialConnectionsTheme.colors.textSecondary
-            )
-            Text(
-                text = email,
-                maxLines = 1,
-                overflow = TextOverflow.Ellipsis,
-                style = FinancialConnectionsTheme.typography.body,
-                color = FinancialConnectionsTheme.colors.textPrimary
+                text = email.getOrElse(0) { 'A' }.uppercaseChar().toString(),
```

Comment:
Can the email address ever be empty?

Reply:
should not happen, but added this as a safety measure : )